Lib/codeop.py

```diff
@@ -102,6 +100,8 @@ def _maybe_compile(compiler, source, filename, symbol):
     try:
         if not code1 and _is_syntax_error(err1, err2):
             raise err1
+        else:
+            return None
```

Comment:
```suggestion
        return None
```

Reply:
Thanks for reviewing.  This choice is a matter of personal preference.  I thought of leaving out 'else', but prefer it here.